components/script/dom/request.rs

```diff
@@ -14,7 +14,7 @@ use dom::bindings::codegen::Bindings::RequestBinding::RequestMethods;
 use dom::bindings::codegen::Bindings::RequestBinding::RequestMode;
 use dom::bindings::codegen::Bindings::RequestBinding::RequestRedirect;
 use dom::bindings::codegen::Bindings::RequestBinding::RequestType;
-use dom::bindings::codegen::UnionTypes::HeadersOrByteStringSequenceSequence;
+use dom::bindings::codegen::UnionTypes::HeadersOrByteStringSequenceSequenceOrByteStringMozMap as HeadersInitType;
```

Comment:
Hmm... `dom::bindings::codegen::Bindings::RequestBinding` should re-export this union type as `HeadersInit`. Have you tried importing that instead?